packages/service-content/src/lib/resources/services/get-newsletter.ts

```diff
@@ -0,0 +1,30 @@
+import { firstRow } from '@blms/database';
+import type { GetNewsletterResponse } from '@blms/types';
+
+import type { Dependencies } from '../../dependencies.js';
+import { computeAssetCdnUrl } from '../../utils.js';
```

Comment:
You might need to remove that and do it frontend instead, check with @max-lt #805